projects/frank-config-layout/src/lib/model/error-flow.ts

```diff
@@ -78,18 +78,18 @@ function transformNode(n: MermaidNode): OriginalNode {
   }
 }
 
-function transformEdge(from: OriginalNode, to: OriginalNode, text: Text): OriginalEdge {
+function transformEdge(from: OriginalNode, to: OriginalNode, text: EdgeText): OriginalEdge {
```

Comment:
```suggestion
function transformEdge(from: OriginalNode, to: OriginalNode, edgeText: EdgeText): OriginalEdge {
```

This could make lines 88 to 92 a bit clearer